lib/ui/elements/toolbar_el.mjs

```diff
@@ -17,6 +18,18 @@ function viewport(dataview) {
 
 }
 
+function layerfilter(dataview) {
+
+  return mapp.utils.html`
+    <button class=${`flat ${dataview.queryparams.filter && 'active' ||''}`}
```

Comment:
Easier to read this way?:
dataview.queryparams.filter ? 'active' : ''

Reply:
I don't mind. Feel free to change. I find logical and ternary operator equally easy to read.